src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write code here
+const employeeList = document.querySelector('body ul');
+
+const strSalaryToNumber = (str) => Number(str.substring(1).replace(',', ''));
+
+function sortList(list) {
+  const nodes = [...list.children];
+
+  nodes.forEach((el) => el.remove());
+
+  const sorted = nodes.sort((first, second) => {
+    return (
+      strSalaryToNumber(second.dataset.salary) -
+      strSalaryToNumber(first.dataset.salary)
+    );
+  });
+
+  list.append(...sorted);
+}
+
+function getEmployees(list) {
+  return [...list.children]
```

Comment:
The `getEmployees` function filters and maps children elements. Ensure that all elements have the necessary `data-*` attributes, or this function may not behave as expected.